unit_tests/tests/trigger/resources/nb2-cranking-good.csv

```diff
@@ -481,10 +481,6 @@ Time [s],crank,cam
 6.940991417,0,1
 6.941894250,1,1
 6.959307333,0,1
-6.959354333,1,1
```

Comment:
this is a blip of noise picked up by the logic analyzer from the ignition system